udata/core/dataset/forms.py

```diff
@@ -79,6 +80,12 @@ class BaseResourceForm(ModelForm):
         _('Publication date'),
         description=_('The publication date of the resource'))
     extras = fields.ExtrasField()
+    schema = fields.SelectField(
+        _('Schema'),
+        choices=[schema['id'] for schema in ResourceSchema.objects()],
```

Comment:
Do I need to explicitly add a `None` to the list? Tests pass without it.

Reply:
No I don't think so.